crates/red_knot_test/src/lib.rs

```diff
@@ -54,6 +61,15 @@ pub fn run(path: &Path, long_title: &str, short_title: &str) {
                     }
                 }
             }
+
+            println!(
+                "\nTo rerun this specific test, add the environment variable: {MDTEST_FILE_FILTER}=\"{}\"",
+                test.name()
+            );
+            println!(
+                "{MDTEST_FILE_FILTER}=\"{}\" cargo test -p red_knot_python_semantic --test mdtest",
+                test.name()
+            );
```

Comment:
maybe this printing is too prescriptive of the exact command to be run / not windows-friendly enough? happy to reword, though i think keeping at least the print for the env var name is useful and solves the discoverability problem

Reply:
<img width="975" alt="Captura de pantalla 2024-11-29 a la(s) 1 57 31 a m" src="https://github.com/user-attachments/assets/5d09d03f-3361-4a25-b574-1889eb9e0a2f">
This is what it looks like rendered

Reply:
I think it's fine as it is now. 

It might be possible to use the [`module_path`](https://doc.rust-lang.org/std/macro.module_path.html) macro in the test function that you then pass through to the run call